Makefile

```diff
@@ -312,7 +320,7 @@ verify-manifests: manifests $(YQ) ## Verify manifests update.
 	yq e -e '.metadata.annotations.containerImage == "$(OPERATOR_IMAGE)"' config/manifests/bases/authorino-operator.clusterserviceversion.yaml
```

Comment:
Because `bundle/manifests/authorino-operator.clusterserviceversion.yaml` is referred a few times in the Makefile, wdyt about declaring a constant for it?

Reply:
Yep I like this suggestion